app/javascript/template_builder/builder.vue

```diff
@@ -699,6 +699,15 @@ export default {
       }
     }
 
+    // Update old party names to new translations
+    this.template.submitters.forEach((submitter) => {
```

Comment:
This is needed so that old forms get the correct naming too. New forms created after this PR is live won't need this.

Reply:
Fair enough, but what old forms? It's all test forms right? If we just deleted all those forms, we wouldn't need this function as far as I can see 🤔

Reply:
I guess if everything in Production is just test forms, I can remove this.